localisation/00_zuwc_united_localisation.csv

```diff
@@ -2884,6 +2866,38 @@ EVTOPTA_WCUND_9915;Bow to the Death God!;;;;;;;;;;;;;x
 EVTOPTB_WCUND_9915;I laugh in the face of death!;;;;;;;;;;;;;x
 EVTDESC_WCUND_9920;With each day that passes on your journey to the heart of Northrend, the blizzards are getting stronger. Finally, you decide to turn back home. A few of your companions have vanished, so this adventure seems to have been a total failure.;;;;;;;;;;;;;x
 dwarven_gnomish_alliance_origin_description;Historical Friend;;;;;;;;;;;;;x
+starts_training_tooltip;�Y[This.GetBestName]�! starts training;;;;;;;;;;;;;x
+ACCEPTANCE;Acceptance;;;;;;;;;;;;;x
+ask_to_be_mentor;Ask to be Mentor;;;;;;;;;;;;;x
+ask_to_be_mentor_desc;�Y[Root.GetBestName]�! is a master of [Root.GetClassSkill]. You can ask [Root.GetHerHim] to mentor the selected character.;;;;;;;;;;;;;x
+EVTDESC_WCCLS_1500_yourself;[mentor.GetBestName] is a well-known master of [mentor.GetClassSkill]. You could ask [mentor.GetHerHim] to train you and share [mentor.GetHerHis] experience. You only need to choose a direction and wait for a letter.;;;;;;;;;;;;;x
+EVTDESC_WCCLS_1500_others;[mentor.GetBestName] is a well-known master of [mentor.GetClassSkill]. You could ask [mentor.GetHerHim] to train [student.GetDynName] and share [mentor.GetHerHis] experience. You only need to choose a direction and wait for a letter.;;;;;;;;;;;;;x
+EVTDESC_WCCLS_2100_yourself;[Root.PrimaryTitle.GetFOA], I have heard legends of your skill. If you are amenable to being my mentor, and do not believe me to be unworthy of your attention, I would be honoured to become your student.\n\nSigned,\n[From.GetTitledName];;;;;;;;;;;;;x
+EVTDESC_WCCLS_2100_others;[Root.PrimaryTitle.GetFOA], I have heard legends of your skill. If you are amenable to being a mentor of [student.GetDynName], and do not believe [student.GetHerHim] to be unworthy of your attention, [student.GetSheHe] would be honoured to become your student.\n\nSigned,\n[From.GetTitledName];;;;;;;;;;;;;x
+EVTOPTA_WCCLS_2100;I'll help [student.GetHerHim].;;;;;;;;;;;;;x
+EVTOPTC_WCCLS_2100;No, [student.GetSheHe] isn't worth my attention.;;;;;;;;;;;;;x
+EVTDESC_WCCLS_2101_yourself;[Root.PrimaryTitle.GetFOA], I gladly agree to be your mentor.\n\nSigned,\n[From.GetTitledName];;;;;;;;;;;;;x
+EVTDESC_WCCLS_2101_others;[Root.PrimaryTitle.GetFOA], I gladly agree to be a mentor of [student.GetDynName].\n\nSigned,\n[From.GetTitledName];;;;;;;;;;;;;x
+EVTDESC_WCCLS_2103_yourself;[Root.PrimaryTitle.GetFOA], apologies, but I have no desire to be your mentor.\n\nSigned,\n[From.GetTitledName];;;;;;;;;;;;;x
+EVTDESC_WCCLS_2103_others;[Root.PrimaryTitle.GetFOA], apologies, but I have no desire be a mentor of [student.GetDynName].\n\nSigned,\n[From.GetTitledName];;;;;;;;;;;;;x
+has_not_students_tooltip;Does NOT have students;;;;;;;;;;;;;x
+EVTDESC_WCCLS_2151;You have waited so long, but have no answer from [mentor.GetBestName]. You don't know what happend, but it seems you need to look for another mentor.;;;;;;;;;;;;;x
+GetClassSkill_the_bow;the bow;;;;;;;;;;;;;x
```

Comment:
> GetClassSkill_the_bow;the bow;;;;;;;;;;;;;x

Bow is not the only thing hunters can use, dwarves use rifles, for example.

mb replace it with "the hunt"?

Reply:
ranged weapon(ry)?

Reply:
Ranged weapon sounds too long.
I would rather base it on your culture... or techs.

Reply:
> Ranged weapon sounds too long.
> I would rather base it on your culture... or techs.

Done it in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/948/commits/1cba1705a2951e9e35eb58086958ed9623390762.

Reply:
Tbh, I would prefer if it was the same for all hunters (marksmanship?) but ok